third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -239,23 +227,23 @@ impl TestConfig {
                 dump_for_only_some_stages: None,
             }
         } else if path.contains("/copy-propagation/") {
+            pipeline.add_processor(Box::new(LiveVarAnalysisProcessor {}));
+            pipeline.add_processor(Box::new(ReferenceSafetyProcessor {}));
+            pipeline.add_processor(Box::new(ExitStateAnalysisProcessor {}));
+            pipeline.add_processor(Box::new(AbilityProcessor {}));
             pipeline.add_processor(Box::new(AvailCopiesAnalysisProcessor {})); // 0
```

Comment:
```suggestion
            pipeline.add_processor(Box::new(AvailCopiesAnalysisProcessor {})); // 4
```

Reply:
@vineethk You know, this mechanism would be less risky if you (1) added a parameter to `add_processor` for the index, and (2) check that you wind up with a contiguous set.

Reply:
@brmataptos Instead of manually adding index and then checking for contiguity, we could just have a boolean param to indicate whether or not to dump based on this processor. I will follow up with a refactor after these PRs land.